app/src/main/res/values-es/strings.xml

```diff
@@ -1929,6 +1929,10 @@
     <string moz:removedIn="94" name="quick_settings_sheet_secure_connection" tools:ignore="UnusedResources">Conexión segura</string>
     <!-- Label that indicates a site is using a insecure connection -->
     <string moz:removedIn="94" name="quick_settings_sheet_insecure_connection" tools:ignore="UnusedResources">Conexión insegura</string>
+    <!-- Label to clear site data -->
+    <string name="clear_site_data">Limpiar cookies y datos del sitio</string>
+    <!-- Confirmation message for a dialog confirming if the user wants to delete all data for current site -->
+    <string name="confirm_clear_site_data">¿Seguro que quieres eliminar las cookies y los datos del sitio para <b>%s</b>?</string>
```

Comment:
Thanks for the translations, but we only update the default (`app/src/main/res/values/strings.xml`) string file as the other translations we get them from some [automatization that come from translation team](https://github.com/mozilla-mobile/fenix/pull/22639). Sorry but we will need to exclude this change as it could break the automatization.

Reply:
No problem :slightly_smiling_face: Removed.

Reply:
Thanks!